src/target/target-opening-hours/TargetOpeningHours.scss

```diff
@@ -0,0 +1,76 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.target-opening-hours-header {
+  height: $spacing-layout-l;
+  background-color: $color-coat-of-arms-blue;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: auto;
+  padding-left: $spacing-m;
+  padding-right: $spacing-m;
+}
+
+.opening-periods-info-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.opening-periods-title {
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+  font-size: $fontsize-heading-m;
+  color: white;
```

Comment:
I usually prefer refactoring when there is the need for it, rather than too early. I feel that the header could be a common component, but would probably extract it when we build another section that needs a header

Reply:
I understand the point of this. 
But in the same time it would be nice to see the whole app's color palette in the same place

Reply:
Yeah I can see that desire for sure, mm.. maybe I'll put the color in to the variables.scss